### PR TITLE
Update instructions NPM, Canary and Teams Channel

### PR DESCRIPTION
**Pull request by rido-min**
This PR updates the BugBash instructions, once is approved and merged in `team-review` branch I will merge it in `master`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
 - Commands (Using the Hub Direct Methods feature)
 - Telemetry (processed through EventHub)
 
-Review the docs [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=release-preview-refresh-iot-pnp).
+Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114283)
```
Comment (rido-min) [inline README.md:15]: @miagdp  is this the right branch for staging docs?

File: README.md [hunk 2]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
...
 
-### Private environments, and preview SDKs
+### BugBash support and feedback
 
-As the time of writing the version of IoT Hub supporting PnP BugBash 2, 08 May2020 release is only deployed to a private hub instance:
+- The teams channel [PnP Public Preview BugBash](https://teams.microsoft.com/l/channel/19%3a0b9d0f166a3d41c69ce90fcca7631962%40thread.skype/PnP%2520Public%2520Preview?groupId=dcc1ac84-f476-4c96-8034-b2d77e54c8bf&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47). Will have team members answering questions during the BugBash.
```
Comment (rido-min) [inline README.md:18]: @miagdp can we use this teams channel?

File: README.md [hunk 3]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
...
+- If you have bugs or feature request, please use this [Bug Template](https://msazure.visualstudio.com/One/_workitems/create/Bug?templateId=a05dbe62-eb63-4028-9ce0-252c552872f3&ownerId=1e65a829-00c0-4dc9-8088-d41678a0d033). You can query existing bugs in this [PnP BugBash Query](https://msazure.visualstudio.com/One/_queries/query-edit/71cca2fe-63bd-4916-84f6-937f0dfaa698/?action=new)
+- For feedback related to the docs, you can use this [azure docs PR](https://github.com/MicrosoftDocs/azure-docs-pr/pull/114283)
```
Comment (rido-min) [inline README.md:20]: @miagdp  do you want to use a new tag in the bug template/query?

File: README.md [hunk 4]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
...
-<Connection Strings for IoT Hub>
-```
+
+## Required environments, SDKs and tools
+
+### IoT Hub
+
+The required hub version is only  available in the canary environment, to create a new hub follow these steps:
+
+- Request access to the `IOTPNP_TEST_BY_MAIN` subscription, via the Teams channel
```
Comment (rido-min) [inline README.md:29]: @minune-msft can you review if these instructions are enough to create hubs in canary?

File: README.md [hunk 5]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
...
 
 This release is supported by the next SDK versions
 
-Device SDK
+##### Device SDK
 
-- azure-iot-sdk-c [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp) branch
-- azure-iot-sdk-node [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp) branch
-- azure-iot-sdk-python [digitaltwins-preview](https://github.com/Azure/azure-iot-sdk-python/tree/digitaltwins-preview) branch
+- azure-iot-sdk-c [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview) branch
```
Comment (rido-min) [inline README.md:40]: @ericmitt  can you confirm this is the right branch? it's not the same I see in the docs

File: README.md [hunk 6]
```diff
@@ -170,17 +178,23 @@ This is an application that connects to IoT Hub using the service SDK offering t
...
 
-### 1. Review the existing samples
+### 1. Review the existing samples and docs
+
+If you need to get familiar with PnP concepts, take a time to familiarize yourself by reading the staging docs available in [review.docs.microsoft.com/azure/iot-pnp](https://review.docs.microsoft.com/azure/iot-pnp?branch=pr-en-us-114283)
+
+To learn how to create a PnP device we have tutorials for C, Node and Python:
 
-Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application running on Linux or Windows to IoT Hub (C)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-c?branch=release-preview-refresh-iot-pnp) to build and run the existing [C device sample](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp/digitaltwin_client/samples)
+Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application running on Linux or Windows to IoT Hub (C)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-c?branch=pr-en-us-114283) to build and run the existing [C device sample](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp/digitaltwin_client/samples)
```
Comment (rido-min) [inline README.md:187]: @ericmitt note I've updated the branch from the doc links

File: README.md [hunk 7]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
...
 
-Device SDK
+##### Device SDK
 
-- azure-iot-sdk-c [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp) branch
-- azure-iot-sdk-node [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp) branch
-- azure-iot-sdk-python [digitaltwins-preview](https://github.com/Azure/azure-iot-sdk-python/tree/digitaltwins-preview) branch
+- azure-iot-sdk-c [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview) branch
+- NPM package for node [azure-iot-digitaltwins-device@1.0.0-pnp-refresh.0](https://www.npmjs.com/package/azure-iot-digitaltwins-device/v/1.0.0-pnp-refresh.0)
+- azure-iot-device [TBD](https://pipy.com/) pip package
```
Comment (rido-min) [inline README.md:42]: @ericmitt do we have instructions to get/install the private python package ? Any chances to publish to pipy?

File: README.md [hunk 8]
```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
...
 
-Service SDK
+##### Service SDK
 
-- azure-iot-sdk-node [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp) branch
-- azure-iot-sdk-python [digitaltwins-preview](https://github.com/Azure/azure-iot-sdk-python/tree/digitaltwins-preview) branch
+- NPM package for node [azure-iot-digitaltwins-service@1.0.0-pnp-refresh.0](https://www.npmjs.com/package/azure-iot-digitaltwins-service/v/1.0.0-pnp-refresh.0)
+- PIP package for python TBD
```
Comment (rido-min) [inline README.md:47]: @ericmitt same for Python service package

File: README.md (hunk 5 above)
Comment (ericmitt) [reply README.md]: will be for PPR, but updated for BugBash

Comment (ericmitt) [reply README.md]: fixed

File: README.md (hunk 7 above)
Comment (ericmitt) [reply README.md:42]: for bugbash no in pipy (will be for PPR), updated the link to the private package

File: README.md (hunk 8 above)
Comment (ericmitt) [reply README.md:47]: fixed

File: README.md (hunk 6 above)
Comment (ericmitt) [reply README.md:187]: great

File: README.md (hunk 1 above)
Comment (ericmitt) [reply README.md:15]: THis link does work, normal? the previous still working

File: README.md (hunk 3 above)
Comment (ericmitt) [reply README.md:20]: LGTM

File: README.md (hunk 2 above)
Comment (ericmitt) [reply README.md:18]: LGTM, using the same as previous, less confusing

**Pull request merged**